### DISCUSSION TITLE
What's the process to debug why some windows are not handled properly?

### DISCUSSION THREAD

**Discussion by Mikkolehtimaki** [glzr-io/glazewm#148] [Q&A] [answered]
Really enjoying this software, great work :) I'm using Power BI Desktop but for some reason it is not detected by Glaze. It opens as a floating window and I seem to have no control of it. So what is the process to have Glaze control this window? I suppose I need to find the "process name" for the configs, what else?

I'm not a Windows power user so things such as figuring the process name are not obvious to me :)

**Comment by maphew** [comment, marked as answer]
AutoIt Info seems to be helpful for identifying window Titles and Class names. However for me GlazeWM worked for Power BI Desktop without having to do anything.
https://www.autoitscript.com/autoit3/docs/intro/au3spy.htm

![image](https://user-images.githubusercontent.com/486200/206570739-a07ab3bd-9d50-4851-a1a7-beb4e4dea469.png)
